releases/release-7.1.4.md

```diff
@@ -0,0 +1,186 @@
+---
+title: TiDB 7.1.4 Release Notes
+summary: 了解 TiDB 7.1.4 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 7.1.4 Release Notes
+
+发版日期：2024 年 x 月 x 日
+
+TiDB 版本：7.1.4
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v7.1/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v7.1/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v7.1.4#version-list)
+
+## 兼容性变更 <!--tw@qiancai 1 条 -->
+
+- 为减少日志打印的开销，TiFlash 配置项 `logger.level` 默认值由 `"debug"` 改为 `"info"` [#8641](https://github.com/pingcap/tiflash/issues/8641) @[JaySon-Huang](https://github.com/JaySon-Huang)
+- (dup): release-6.5.8.md > 兼容性变更 - 新增 TiKV 配置项 [`gc.num-threads`](https://docs.pingcap.com/zh/tidb/v6.5/tikv-configuration-file#num-threads-从-v658-版本开始引入)，用于设置当 `enable-compaction-filter` 为 `false` 时 GC 的线程个数 [#16101](https://github.com/tikv/tikv/issues/16101) @[tonyxuqqi](https://github.com/tonyxuqqi)
+
+## 改进提升
+
++ TiDB <!--tw@Oreoxmt 1 条 -->
+
+    - (dup): release-7.5.1.md > 改进提升> TiDB - 增强特定情况下 `OUTER JOIN` 转 `INNER JOIN` 的能力 [#49616](https://github.com/pingcap/tidb/issues/49616) @[qw4990](https://github.com/qw4990)
+    - 当设置 `force-init-stats` 为 `true` 时，即 TiDB 启动时等待统计信息初始化完成后再对外提供服务，此时 TiDB 会同时禁止 HTTP 服务器的启动 [#50854](https://github.com/pingcap/tidb/issues/50854) @[hawkingrei](https://github.com/hawkingrei)
```

Comment:
https://github.com/pingcap/tidb/issues/50854: In the current implementation, the force_init_stats block the startup of HTTP APIs. This could potentially lead to issues as the API cannot be accessed before the stats are properly initialized.

Reply:
@elsa0520 麻烦重新修改下这条 release notes